java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -1335,4 +1336,28 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_nansToNulls(JNIEnv *env
   CATCH_STD(env, 0)
 }
 
+JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_isFloat(JNIEnv *env, jobject j_object, jlong handle) {
+
+  JNI_NULL_CHECK(env, handle, "native view handle is null", 0)
+
+  try {
+    cudf::column_view * view = reinterpret_cast<cudf::column_view *>(handle);
```

Comment:
Missing call to cudf::jni::auto_set_device here.
```suggestion
    cudf::jni::auto_set_device(env);
    cudf::column_view * view = reinterpret_cast<cudf::column_view *>(handle);
```